roles/ceilometer-data/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: install ceilometer compute service
+  upstart_service: name=ceilometer-agent-compute
+                   user=ceilometer
+                   cmd=/usr/local/bin/ceilometer-agent-compute
+  notify:
+    - restart ceilometer services
+
+- name: pip install libvirt-python in venv on compute
+  pip: name=libvirt-python virtualenv="{{ ceilometer.virtualenv }}"
+  notify:
+    - restart ceilometer services
```

Comment:
I made it to install libvirt-python as part of a source install on all three nodes, and added in a when conditional here to only do on a package install (until ceilometer gets merged into giftwrap)